src/components/star-rating/star-rating.react.js

```diff
@@ -8,14 +8,8 @@ import {UIBase} from '../layout';
 
 import styles from './star-rating.css';
 
-type TProps = {
-  score: Number,
-  component?: string,
-  className?: string | Object,
-};
-
-export default function StarRating(props: TProps) {
-  const {score = 0, className, ...otherProps} = props;
+function StarRating(props) {
+  const {score = 0, className, color, ...otherProps} = props;
```

Comment:
This should have a default for the `color` prop.

Reply:
Good idea, at the moment the colour is defined in the css, but actually makes more sense having it centralised in the component 👍

Reply:
Done ;)